src/configure.ts

```diff
@@ -16,13 +18,49 @@ export function jestPreviewConfigure(
 
   options.externalCss.forEach((cssFile) => {
     const basename = path.basename(cssFile);
+
     // Avoid name collision
     // Add `global` to let `jest-preview` server that we want to cache those files
     const destinationBasename = `cache-${basename}`;
-    const destinationFile = path.join(
-      './node_modules/.cache/jest-preview-dom',
-      destinationBasename,
-    );
+    const destinationFile = path.join(CACHE_FOLDER, destinationBasename);
+
+    // If sass file is passed, we need to transform it to css
+    if (cssFile.endsWith('.scss') || cssFile.endsWith('.sass')) {
+      const sass = require('sass');
+
+      // Create cache folder if it doesn't exist
+      if (!fs.existsSync(CACHE_FOLDER)) {
+        fs.mkdirSync(CACHE_FOLDER, {
+          recursive: true,
+        });
+      }
+
+      let sassLoadPaths: string[] = [];
+      // Save sassLoadPaths to cache, so we can use it in the transformer
+      if (options.sassLoadPaths) {
+        sassLoadPaths = options.sassLoadPaths.map(
+          (path) => `${process.cwd()}/${path}`,
+        );
+
+        fs.writeFileSync(
+          path.join(CACHE_FOLDER, SASS_LOAD_PATHS_CONFIG),
+          JSON.stringify(sassLoadPaths),
+        );
+      }
+
+      // Transform sass to css
+      const sassDestinationFile = destinationFile.replace(
+        /\.(scss|sass)$/,
+        '.css',
+      );
```

Comment:
I think move this declaration next to `fs.writeFileSync(sassDestinationFile, sassResult.css);` is more readable